### PR TITLE
Add user get function

### PR DESCRIPTION
**Pull request by ovekyc**
- Add a user function
- Add testcode
- Edit test sequence

### CODE REVIEW

File: src/server/users/router.js [hunk 1]
```diff
@@ -3,4 +3,30 @@ import controller from './controllers';
...
+ *
+ * @apiSuccess {Object} json
+ *
+ * @apiSuccessExample Success-Response:
+ *     HTTP/1.1 200 OK
+ *     {
+ *        key: 'user-unique-key',
+ *        name: 'test-user',
+ *        email: 'test@email.com',
+ *        imagePath: 'url-of-image'
```
Comment (hewonjeong) [inline src/server/users/router.js]: how about imageUrl?

**Comment by hewonjeong** [comment]
lgtm

**Comment by festinalente91** [comment]
lgtm

**Comment by ovekyc** [comment]
lgtm

**Pull request merged**